students/toogii/lesson03/mailroom.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+from time import sleep
+
+
+choices = ('Send a Thank You', 'Create Report')
+donations = {'TOOGII DASHDAVAA': {"donation": 3, "donation_amnt": 30000},'MARK ZACHERBERG': {"donation": 1, "donation_amnt": 1000}, 'JEFF BEZOS': {"donation": 3, "donation_amnt": 15000}, 'BILL GATES': {"donation": 1, "donation_amnt": 500}, 'LARRY PAGE': {"donation": 2, "donation_amnt": 10000}}
```

Comment:
The idea is to have list of donations and to calculate the sum of donations and the average amount.